### DISCUSSION TITLE
归档（archives）页如何每页显示更多项？

### DISCUSSION THREAD

**Discussion by scutLMG** [ppoffice/hexo-theme-icarus#998] [Q&A] [answered]
很久没用了，找不到配置的地方，翻了下文档好像也没找到。
官方的页面，一页可以显示很多项
![image](https://user-images.githubusercontent.com/45113495/147578924-cde18e2d-2b37-4588-9ca7-295aa56d5e1c.png)
我的
![image](https://user-images.githubusercontent.com/45113495/147579035-196c50c2-6aab-4d4e-b108-e79407079daa.png)
是因为我每个都有缩略图还是什么原因呢？
我想每页显示更多，但在各个配置文件里没有找到类似per_page的archives配置，应该怎么弄呢？

**Comment by ppoffice** [comment, marked as answer]
https://hexo.io/zh-cn/docs/configuration.html#%E5%88%86%E9%A1%B5

**Reply by scutLMG** [reply]
感谢，解决了！我以为那个是主页文章数，和其他页无关呢。感谢感谢！
